### PR TITLE
Tidy dependencies

### PR DESCRIPTION
**Pull request by insacuri**
## What? and Why?
This just upgrades dependencies as they've not been looked at for quite a while.  A few of them caused tests to fail which is why there are a few minor code changes

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: console/__init__.py [hunk 1]
```diff
@@ -32,8 +32,8 @@
     ftp = FTP(settings.FTP_HOST)
     ftp.login(user=settings.FTP_USER, passwd=settings.FTP_PASS)
     len([fname for fname, fmeta in ftp.mlsd(path=settings.SDX_FTP_DATA_PATH)])
-except Exception as e:
-    logger.info("MLSD command not availible in the FTP server, will use LIST instead", exception=e)
+except Exception:
+    logger.exception("MLSD command not availible in the FTP server, will use LIST instead")
```
Comment (owencjones) [inline console/__init__.py:36]: Why remove the exception?  Will it not be useful in finding issues.

**Review by owencjones — APPROVED** [review]
LGTM, but consider the comment. 👍

File: console/__init__.py (hunk 1 above)
Comment (jcox-dev) [reply console/__init__.py:36]: @owencjones `logger.exception(...` includes a stacktrace so the variable isn't required 👍

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**